modules/core/src/v2/coins/eth.ts

```diff
@@ -207,11 +219,8 @@ export interface RecoverOptions {
   krsProvider?: string;
   gasPrice?: number;
   gasLimit?: number;
-  eip1559?: { maxPriorityFeePerGas: number; maxFeePerGas: number };
-  replayProtectionOptions?: {
-    chain: string | number;
-    hardfork: string;
-  };
+  eip1559?: EIP1599;
+  replayProtectionOptions?: ReplayProtectionOptions;
```

Comment:
Can we update BuildTransactionParams on line 226 to reflect and standardize the use of the newly added interfaces?

Reply:
Good catch @mmcshinsky-bitgo , Updated it.